tyxe/priors.py

```diff
@@ -189,8 +192,12 @@ def __init__(self, prior_dict, *args, **kwargs):
         self.prior_dict = prior_dict
 
     def prior_dist(self, name, module, param):
-        return self.prior_dict[name]
-
+        # I find this preferable to the hack in update_ as it gets to the root. 
```

Comment:
Unless the the naming of the keys can be fixed in pyro, this strikes me as the nicest solution. Rather than the hack in line 142